packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -249,6 +249,10 @@ function getEmptyStateValue() {
   return testRenderer._instance.state;
 }
 
+function isMemo(node) {
+  return typeof node.type === 'object' && node.type.$$typeof === Symbol.for('react.memo');
```

Comment:
This must not use `Symbol`; instead, can you import `REACT_MEMO_TYPE` from `react-is`, and compare to that? Specifically:
```suggestion
  return node.type && node.type.$$typeof === REACT_MEMO_TYPE;
```

Separately, can you file an issue on `react-is` to add an `isMemo` function?